ddl/ddl_api.go

```diff
@@ -4904,6 +4904,15 @@ func isDroppableColumn(tblInfo *model.TableInfo, colName model.CIStr) error {
 		return ErrCantRemoveAllFields.GenWithStack("can't drop only column %s in table %s",
 			colName, tblInfo.Name)
 	}
+	visibleColumCnt := 0
+	for _, column := range tblInfo.Columns {
+		if !column.Hidden {
+			visibleColumCnt++
+		}
+	}
+	if visibleColumCnt == 1 {
```

Comment:
What if `alter table t drop column a, drop column b`?

Reply:
drop multiple columns is not checked correctly, I will fix it.